frontend/src/components/nodes/NodeRow.tsx

```diff
@@ -12,6 +12,15 @@ interface Props {
 export default class extends React.PureComponent<Props> {
   render() {
     const { node } = this.props;
+    let statusIdentifier = new Map([
```

Comment:
Let's move this to be a static field on the component class, we don't need to recreate it on every render.